packages/extension-browser/src/devtools/utils/storage.ts

```diff
@@ -1,6 +1,30 @@
+let storage: {
+    getItem(key: string): string | null;
+    setItem(key: string, value: string): void;
+};
```

Comment:
Let's pull the signatures from the built-in `Storage` type, but still scoped to just the methods we want.
```suggestion
let storage: Pick<Storage, 'getItem' | 'setItem'>;
```

Reply:
Didn't know `Pick` before. This is awesome.